apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -291,7 +295,14 @@ protected TableMetadata mapTableResponseToTableMetadata(GetTableResponseBody res
             .dbName(responseBody.getDatabaseId())
             .tableName(responseBody.getTableId())
             .build();
-    String creator = getTable(metadata).getTableCreator();
+    GetTableResponseBody tableResponseBody = getTable(metadata);
+
+    if (tableResponseBody == null) {
+      log.error("Error while fetching metadata for table: {}", metadata);
+      return null;
```

Comment:
```suggestion
      return TableMetadata;
```

Reply:
Not following this.The change is to return null and skip that as part of results aggregation so prefer to return null here

Reply:
`tableResponseBody` is already null here.

Reply:
Makes sense now. I think suggestion was to return tableMetadata before edit. Prefer to return explicitly  as null.